databases/week2/Week2-homework.sql

```diff
@@ -0,0 +1,41 @@
+USE `HYF_Lesson_2`;
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT 
+    user.email, task.title, task.description
+FROM
+    (task JOIN user_task ON task.id = user_task.task_id)
+        JOIN
+    user ON user.id = user_task.user_id
+WHERE
+    email LIKE '%@spotify.com%';
+
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT 
+    user.name, task.title, task.status_id
+FROM
+    (task JOIN user_task ON task.id = user_task.task_id)
+        JOIN
+    user ON user.id = user_task.user_id
+WHERE
+    name LIKE '%Donald Duck%'
+        AND task.status_id =
+        (SELECT id FROM status WHERE status.name = 'Not Started');
```

Comment:
Here I need to see the status of the task.  You only know what status-id=1 is meant.

How about if you try like below?

```
SELECT task.id, task.title AS 'Task Name', user.name AS 'user', status.name AS status
FROM user_task
JOIN task ON user_task.task_id = task.id
JOIN user ON user_task.user_id = user.id
JOIN status ON task.status_id = status.id
WHERE status.name = "Not started" AND user.name = 'Donald Duck';
```